src/server.rs

```diff
@@ -569,12 +575,14 @@ pub extern "C" fn rustls_server_session_get_negotiated_ciphersuite(
 /// the SignatureSchemes supported by the client.
 struct ResolvesServerCertFromChoices {
     choices: Vec<Arc<CertifiedKey>>,
+    ocsp: OcspProvider,
 }
 
 impl ResolvesServerCertFromChoices {
-    pub fn new(choices: &[Arc<CertifiedKey>]) -> Self {
+    pub fn new(choices: &[Arc<CertifiedKey>], ocsp: OcspProvider) -> Self {
         ResolvesServerCertFromChoices {
             choices: Vec::from(choices),
+            ocsp: ocsp,
```

Comment:
```suggestion
            ocsp,
```
Since the variable name and the struct field match, you can use this form. I think `cargo clippy` will complain if you don't.